lib/puppet/modulebuilder/builder.rb

```diff
@@ -251,20 +250,11 @@ def build_package
     def ignored_files
       require 'pathspec'
 
-      @ignored_files ||=
-        begin
-          ignored = if ignore_file.nil?
-                      PathSpec.new
-                    else
-                      PathSpec.new(read_file(ignore_file, open_args: 'rb:UTF-8'))
-                    end
-
-          ignored = ignored.add("/#{File.basename(destination)}/") if File.realdirpath(destination).start_with?(File.realdirpath(source))
+      ignored = PathSpec.new
+      ignored = ignored.add("/#{File.basename(destination)}/") if File.realdirpath(destination).start_with?(File.realdirpath(source))
+      ignored = ignored.add(IGNORED.join("\n"))
```

Comment:
Looking at the implementation it can also call `.each` so I'd assume this works:
```suggestion
      ignored.add(IGNORED)
```

And if you add it in the constructor, it's obviously no longer needed:
```suggestion
```